src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/prisonapi/PrisonApiClient.kt

```diff
@@ -52,7 +51,9 @@ class PrisonApiClient(private val prisonApiWebClient: WebClient) {
         ),
       )
       .retrieve()
-      .bodyToMono(Event::class.java)
-      .onErrorResume(WebClientResponseException.NotFound::class.java) { Mono.empty() }
+      .bodyToMono(ScheduledEvent::class.java)
       .block()
 }
+
+// Overriding due to deserialisation issues from generated type.
+data class ScheduledEvent(val eventId: Long)
```

Comment:
Although not ideal I am comfortable with this, we really only need the event ID for logging purposes anyway.